### PR TITLE
set version to 10.3

### PR DESCRIPTION
**Pull request by a4z**
Set new version so #21 will also show up in the JetBrains Marketplace

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,16 @@
 # djinni-intellij-plugin Changelog
 
 ## [Unreleased]
+- nothing
+
+## [0.10.3]
```
Comment (jothepro) [inline CHANGELOG.md]: No, thats wrong 🙈 

1. I doupt that only CLion is affected? Not sure
2. The current changes have to be put into the `[Unreleased]` Block:

```
# djinni-intellij-plugin Changelog

[Unreleased]
- set pluginUntilBuild = 203.* to work with latest IntelliJ Platform

### [0.10.2]
- interfaces can now have `const` methods

...
```

**Review by jothepro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (a4z) [reply CHANGELOG.md]: should be fixed with the latest commit

Comment (jothepro) [reply CHANGELOG.md]: > I doupt that only CLion is affected? Not sure

Can you check that? My understanding is that the whole IntelliJ Platform got an update to 203, not just CLion?

Comment (a4z) [reply CHANGELOG.md]: they are all shown as 203.*  in my tool box, yes

**Comment by jothepro** [comment]
FYI to make the plugin really show up in the marketplace you have to publish the release that will be drafted by the pipeline on merge in `main`. This will make the release pipeline run. 

It's complicated 🙈

File: CHANGELOG.md (hunk 1 above)
Comment (a4z) [reply CHANGELOG.md]: except, if course, Android Studio, this has somehow different version ...

**Review by jothepro — APPROVED** [review]
👍

**Pull request merged**